doc/snippets/EigenIntegration.cpp

```diff
@@ -64,6 +64,24 @@ Debug{} << Eigen::Array3i{1, 42, -3}; // prints   1
 static_cast<void>(d);
 }
 
+{
+/* [IntegrationStridedArrayView] */
+float data[15];
+Containers::StridedArrayView2D<float> view{data, {3,5}};
+
+/* To avoid a copy make sure to provide the correct type or use auto. */
+auto map = EigenIntegration::arrayCast(view);
+
+/* Copy everything into Eigen Matrix. */
```

Comment:
```suggestion
/* Unlike the above, this copies everything */
```

... or something that makes the point clearer, I'm too tired to invent a better comment right now.